### PR TITLE
Add wrappers for formatting exceptions

### PR DESCRIPTION
**Pull request by bwiernik**
Closes https://github.com/easystats/easystats/issues/298

### CODE REVIEW

File: R/format_message.R [hunk 1]
```diff
@@ -81,6 +89,59 @@ format_message <- function(string, ..., line_length = 0.9 * options()$width, ind
...
+#'   format_alert("This is a warning.", type = "warning")
+#'   format_warning("This is a warning.")
+#'   format_error("This is an error.")
+#' }
+format_alert <- function(string, ..., line_length = 0.9 * getOption("width", 89), indent = "  ", type = c("message", "warning", "error"), call. = FALSE) {
+  type <- match.arg(type, choices = c("message", "warning", "error"))
+  if (type == "message") {
+    message(format_message(
+      string = string, ...,
+      line_length = 0.9 * getOption("width", 89), indent = "  "
```
Comment (strengejacke) [inline R/format_message.R:117]: Shouldn't this be `line_length = line_length`?

**Comment by bwiernik** [comment]
Failing test appears to be unrelated to these changes

File: R/format_message.R [hunk 2]
```diff
@@ -81,6 +89,59 @@ format_message <- function(string, ..., line_length = 0.9 * options()$width, ind
...
+      string = string, ...,
+      line_length = line_length, indent = indent
+    ), call. = call.)
+  }
+}
+
+#' @name format_warning
+#' @rdname format_message
+#' @export
+format_warning <- function(..., type = "warning") {
```
Comment (strengejacke) [inline R/format_message.R:135]: I would probably even hardcode here, i.e. no `type` argument and then call `format_alert(..., type = "warning")`. Same for `format_error()`.

File: R/format_message.R [hunk 3]
```diff
@@ -61,7 +69,7 @@
 #' message(msg)
 #'
 #' @export
-format_message <- function(string, ..., line_length = 0.9 * options()$width, indent = "  ") {
+format_message <- function(string, ..., line_length = 0.9 * getOption("width", 89), indent = "  ") {
```
Comment (strengejacke) [inline R/format_message.R:72]: Why `89` as default? I think the normal default is `80`.

**Review by strengejacke — APPROVED** [review]

**Pull request merged**